### ISSUE TITLE
Results from V1.0.1

### ISSUE THREAD

**Issue by smarley2** [blinken/power-rail-probe#3]
Hi, can we talk about the results you got?

Did you try different values of L in the output of OPA? 
Seems that is highly relevant for the deep you have when change from the Opamp to the capacitor.

What's the limitation factor you think you have to not achieve higher frequencies?
The parasitic inductance would reduce your bandwidht, but your trace on the layout seems to be very short on the AC path.
You think is due to the capacitor parasitic? Did you try different PN of capacitors?

![image](https://github.com/user-attachments/assets/8e1914fc-ff12-473e-b25e-65cec2296b22)

**Comment by blinken** [comment]
> Did you try different values of L in the output of OPA?
> Seems that is highly relevant for the deep you have when change from the Opamp to the capacitor.

Yes, I spent a couple of weeks on this t the beginning of the year. The optimal values are the ones chosen - take a look at https://hachyderm.io/@blinken/111671085003017347 and https://hachyderm.io/@blinken/112859345117485764 (v1.0.1) for a discussion of this. Make sure you're looking at the correct schematic for v1.0.1 (https://kicanvas.org/?github=https%3A%2F%2Fgithub.com%2Fblinken%2Fpower-rail-probe%2Fblob%2Fmain%2Flow-noise-power-probe.kicad_sch) 

I thought as recently as a couple of weeks ago that the dip in S12 at ~750kHz was down to opamp compensation, but I'm increasingly convinced it's due to the inductance of RV1 (the potentiometer). I've tried replace this in the past, with no change, but I think I'd like to look in more detail.

Note that removing C2 improves the frequency response @ 750kHz, so I'm removing it from shipping boards going forward & will update the schematic shortly.

![image](https://github.com/user-attachments/assets/8f56c228-221a-45fc-b304-0a0987491210)


> What's the limitation factor you think you have to not achieve higher frequencies?
> The parasitic inductance would reduce your bandwidht, but your trace on the layout seems to be very short on the AC path.
> You think is due to the capacitor parasitic? Did you try different PN of capacitors?

Yes :)

From simulation I believe the limitation for higher frequency response is the BNC connectors. I'd like to try a dual footprint with SMA on the next board revision to see if it helps. 

I get the same frequency response (falloff @ ~500MHz) in simulation if I insert 10p of capacitance here:

![image](https://github.com/user-attachments/assets/a315e279-8472-452f-b639-b00fe5279c03)

![image](https://github.com/user-attachments/assets/911b7364-2da3-4e65-8689-7a44b82dcd8d)

It might also be something like the interaction between R2 & the connector capacitance forming a low-pass filter, but R2 is needed to damp down the notch filter formed by C1 and L1.

I've added the simulation file under doc/v1.0.1/v1.0.1.TSC if you would like to have a look.

**Comment by smarley2** [comment]
On your measurement, are you considering the cable also, or you calibrated the VNA and connected directly to the device?
Because if you look a low noise cable like LMR195 it will give to you 83.3pF/m (https://www.koax24.de/storage/datasheet/de/050123_Datenblatt_-_LMR-195.pdf)

And how about removing the output resistor that you think is making the filter?
Try something like this and check how is the noise and the response:
![image](https://github.com/user-attachments/assets/f951d811-7b22-4ec0-a135-8a1e8c0cba7a)

**Comment by blinken** [comment]
Sorry to say that I don't have time to work on the probe this week, but please feel free to try and report back on how it looks in simulation! I'd be interested to see what you find.

We use a calibrated HP 8753ES in our lab with these cables: https://docs.rs-online.com/d8ff/0900766b81570f7c.pdf. The VNA calibration is done with a calibrated/traceable cal kit at the tip of the cable.
